domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SampleClaim.java

```diff
@@ -497,7 +517,7 @@ public Claim build() {
             claimData,
             createdAt,
             issuedOn,
-            issuedOn.plusDays(5),
+            issuedOn != null ? issuedOn.plusDays(5) : null,
```

Comment:
issued on should always be there, because it's calculated in the callback

Reply:
my understanding is, issued on will be there when payment is not successful.

Reply:
it will have been populated by initiate payment already

Reply:
done

Reply:
Hi @simad , speaking to @ckalista about this and he wants to double check the reason for issuedOn calculated in initiate payment, is it because we need this to calculate fee?